src/plugins/intel_gpu/src/runtime/ocl/ocl_engine.cpp

```diff
@@ -125,29 +125,29 @@ allocation_type ocl_engine::detect_usm_allocation_type(const void* memory) const
                                        : allocation_type::unknown;
 }
 
-bool ocl_engine::check_allocatable(const layout& layout, allocation_type type) const {
+bool ocl_engine::check_allocatable(const layout& layout, allocation_type type) {
     OPENVINO_ASSERT(supports_allocation(type) || type == allocation_type::cl_mem, "[GPU] Unsupported allocation type: ", type);
     auto used_mem = get_used_device_memory(allocation_type::usm_device) + get_used_device_memory(allocation_type::usm_host);
-#ifdef __unix__
-    // Prevent from being killed by Ooo Killer of Linux
-    OPENVINO_ASSERT(layout.bytes_count() + used_mem <= get_max_memory_size(),
-            "[GPU] Exceeded max size of memory allocation: ",
-            "Required ", layout.bytes_count(), " bytes, already occupied : ", used_mem, " bytes, ",
-            "but available memory size is ", get_max_memory_size(), " bytes");
-#else
+
     if (layout.bytes_count() + used_mem > get_max_memory_size()) {
-        GPU_DEBUG_COUT << "[Warning] [GPU] Exceeded max size of memory allocation: " << "Required " << layout.bytes_count() << " bytes, already occupied : "
+        GPU_DEBUG_COUT << "[GPU] Exceeded max size of memory allocation: " << "Required " << layout.bytes_count() << " bytes, already occupied : "
```

Comment:
This print is shown for both  unix and windows. 
e.g., if unix, first this message is printed and then printed again at the exception throwing. 
Please let the message to be shown only once. 
linux : as a exception 
windows: as a warning

Reply:
Updated. Thanks!